types/react-sketchapp/tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
     "compilerOptions": {
-        "allowSyntheticDefaultImports": true,
```

Comment:
Does this still work? If I remember correctly, if you remove this option then you'd have to change the imports to `import * as React from 'react';` but i'm not entirely sure. Can you confirm that this works?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34637/files#diff-9f40a6b1b02ac40eb8462ac5e4bfbba9R9 e.g.

Reply:
Yes, that's why I added it in the first place, but then Travis threw an error, because of the `no-import-default-of-export-equals` rule, which is also referring to the `allowSyntheticDefaultImports` compile option.

[See docs](https://github.com/Microsoft/dtslint/blob/master/docs/no-import-default-of-export-equals.md)